index.d.ts

```diff
@@ -63,6 +63,7 @@ export interface BootstrapTableColumn {
   checkbox?: boolean;
   halign?: any;
   switchable?: boolean;
+  switchableLabel?: any;
```

Comment:
`any` or `string` ?
Does this method only accept a string or also a function ?

Reply:
Only String; PR updated.